README.md

```diff
@@ -321,26 +262,49 @@ If a test errors but selenium is still working, it will take
 a screenshot of the browser at the point of the error and write it
 as a png file in the project directory.
 
-##### Generating screenshots
+##### Viewing running tests in the browser
+
+Selenium tests run locally will forward port 7000 to the inner container. To
+view the test server with your browser, go to `http://<your_mm_ip_address>:7000`.
+
+
+# Running Commands
+
+#### Setting user state
+
+You'll often need a user to be in a certain state with respect to some course/course run (e.g.: your user needs to have
+a passing grade in some course). There is a management command that can be used to easily accomplish this kind of change 
+of state.
+
+	# See usage details
+    docker-compose run web ./manage.py alter_data --help
+    # See example commands
+    docker-compose run web ./manage.py alter_data examples
+
+#### Generating screenshots
 
-To generate screenshots of various dashboard states, run this command:
+There is a helper script that uses Selenium to create screenshots of various learner dashboard states.
+To generate these screenshots, run this command:
 
     ./scripts/test/run_snapshot_dashboard_states.sh
 
-##### Viewing tests via browser
+#### Validating financial aid info
+
+To validate prices and financial aid discounts for all programs run:
+
+    docker-compose run web ./manage.py validate_db
+    
+Screenshots are saved to the `output` directory.
```

Comment:
is this line supposed to be part of the section above?

Reply:
Right you are!